spec/features/carriers/loans_spec.rb

```diff
@@ -39,4 +39,15 @@
     visit carrier_url(carriers(:unavailable))
     expect(page).not_to have_selector(:link_or_button, "Checkout")
   end
+
+  scenario 'Volunteer is able to see a loan history for a specific user' do
+    sign_in volunteer
+    visit users_url
+
+    click_link member.first_name
+
+    expect(page).to have_content(member.first_name)
+
+    # Need to add a checkout here and check that the checkout is on the history
```

Comment:
@mark100net I need to add tests to cover this and at first I put the test in the user feature spec.  But then I think it belongs here.  But I'm fumbling around a little to make this work.  Any advice?

Reply:
If a feature spec I think it should go in the user feature spec (it is part of a user controller action). TBH I wouldn't be opposed to just adding it to the users request spec (`spec/requests/users_spec.rb`). Either way.

Reply:
I took this out and added a test to the user feature spec.  Maybe the request spec would have worked too....it works 🤷‍♀